server/index.js

```diff
@@ -1,17 +1,19 @@
-//Initializing express, middleware, CORS
+
+//Initializing express, middleware, and CORS
+
 const express = require("express");
 const bodyParser = require("body-parser");
 const cors = require("cors");
 const path = require("path");
 const crypto = require("crypto");
 const mongoose = require("mongoose");
-const multer = require("multer");
-const GridFsStorage = require("multer-gridfs-storage");
-const Grid = require("gridfs-stream");
-const methodOverride = require("method-override");
+const multer = require("multer")
+const GridFsStorage = require("multer-gridfs-storage")
+const Grid = require("gridfs-stream")
+const methodOverride = require("method-override")
```

Comment:
Shouldn't these require semicolons after declaring them?

Reply:
perhaps, I don't seem to be getting an error but I'll look into it